tasmota/tasmota_support/support_esp.ino

```diff
@@ -656,17 +656,14 @@ float CpuTemperature(void) {
   return t;
 */
 #else
-  #ifndef CONFIG_IDF_TARGET_ESP32S3
     // Currently (20210801) repeated calls to temperatureRead() on ESP32C3 and ESP32S2 result in IDF error messages
-    static float t = NAN;
-    if (isnan(t)) {
-      t = (float)temperatureRead();  // In Celsius
+static float t = NAN;
+  if (isnan(t)) {
+    t = (float)temperatureRead();  // In Celsius
     }
     return t;
-  #else
-    return NAN;
   #endif
-#endif
+return NAN;
```

Comment:
Why is this line still here?

If possible pls keep indent spacing logical too...